### DISCUSSION TITLE
Include username attribution in email notifications

### DISCUSSION THREAD

**Discussion by amiryal** [community/community#108365] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Email notifications show only full names in the `From` field, and nowhere in the body of the message does the username appear. To associate email messages with people on the web pages, one has to learn everyone’s usernames and full names at the same time. Currently it involves hovering the mouse pointer (♿) over usernames to reveal the profile cards with full names. It would be a little less mentally taxing if either usernames or full names are used **consistently** to attribute messages to users.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @amiryal! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
